src/common/regions.ts

```diff
@@ -0,0 +1,146 @@
+import { chain, fromPairs } from 'lodash';
+import urlJoin from 'url-join';
+import US_STATE_DATASET from 'components/MapSelectors/datasets/us_states_dataset_01_02_2020.json';
+import countyAdjacencyMsa from './data/county_adjacency_msa.json';
+const { state_dataset, state_county_map_dataset } = US_STATE_DATASET;
+
+export type FipsCode = string;
+
+export enum RegionType {
```

Comment:
It would be great if the values of this could match (or just be) [AggregationLevel](https://github.com/covid-projections/covid-projections/blob/develop/src/api/schema/RegionSummary.d.ts#L26) so it matches the values from the API

Reply:
Good catch, will update!